### PR TITLE
add enviroment config on sentry

### PR DESCRIPTION
**Pull request by thiagoarioli**
make the possibility to add environment on sentry init with .env config

### CODE REVIEW

**Comment by thiagoarioli** [comment]
what do you think about this pr @Perafan18 ?

File: providers/Sentry.js [hunk 1]
```diff
@@ -8,7 +8,11 @@ class SentryProvider extends ServiceProvider {
     const Config = this.app.use('Adonis/Src/Config')
     this.app.singleton('Sentry', () => {
       const sentryDns = Config.get('sentry.dns')
-      Sentry.init({ dsn: sentryDns })
+      const environment = Config.get('sentry.environment')
+      Sentry.init({ 
+        dsn: sentryDns,
+        environment
```
Comment (Perafan18) [inline providers/Sentry.js]: This should be `environment : environment ` or
```
Sentry.init({ 
        dsn: Config.get('sentry.dns'),
        environment: Config.get('sentry.environment')
})
```

Comment (thiagoarioli) [reply providers/Sentry.js]: if you have the same name on key:value you can use only key.
https://eslint.org/docs/rules/object-shorthand

**Comment by thiagoarioli** [comment]
@Perafan18 do you have an idea when will release this?

**Pull request merged**